### PR TITLE
DCOS-60471: Update Docs Site for the release of DC/OS Jenkins Service 3.6.0-2.190.1

### PR DESCRIPTION
**Pull request by kaiwalyajoshi**
## Jira Ticket
[DCOS-60471](https://jira.mesosphere.com/browse/DCOS-60471)

## Description of changes being made
Updates to docs site for release of DC/OS Jenkins 3.6.0-2.190.1

## Checklist
- [x ] Make sure you change all affected versions (e.g. 1.10, 1.11, 1.12, 1.13).
- [ x] Test all commands and procedures where applicable.
- [x ] Add [redirects](https://github.com/mesosphere/dcos-docs-site/wiki/Redirects) if you are moving a page.

See the [contribution guidelines](https://github.com/mesosphere/dcos-docs-site/wiki/Contributing) for more information.

### CODE REVIEW

File: pages/mesosphere/dcos/services/jenkins/3.6.0-2.190.1/index.md [hunk 1]
```diff
@@ -0,0 +1,20 @@
+---
+layout: layout.pug
+navigationTitle: Jenkins 3.5.4-2.150.1
```
Comment (vinodkone) [inline pages/mesosphere/dcos/services/jenkins/3.6.0-2.190.1/index.md]: version is wrong here and below?

Comment (kaiwalyajoshi) [reply pages/mesosphere/dcos/services/jenkins/3.6.0-2.190.1/index.md]: Oof, good catch!

**Comment by ck4adventure** [comment]
Hey @vinodkone and @kaiwalyajoshi . This PR is being submitted from a forked repository and not a clone. Since you are also employees, you have the ability to clone and work more directly. This is necessary because they way you have done this, I cannot pull the files and check the build before merging to staging, which is critical because that is direct to Prod, and we do not have any automated checks on our repo at this time. 
Please take the time to clone our repo and add these edits into a branch so that the entire team can collaborate on them normally.
Thanks!!
Any questions or access issues, please feel free to ping me direct on slack!
cc @cmastr

**Comment by kaiwalyajoshi** [comment]
Hi @ck4adventure 
The contribution guide says to fork the repo: https://github.com/mesosphere/dcos-docs-site/wiki/Contributing#overview

Please update the guide, I'll create a new branch shortly.

**Comment by kaiwalyajoshi** [comment]
Closed in favour of #2608 which mirrors this in the same repo with all the same patches.

**Comment by ck4adventure** [comment]
@kaiwalyajoshi, actually I spent yesterday afternoon doing just that, and will continue removing those references :)

**Pull request closed without merging**